README.md

```diff
@@ -79,37 +61,63 @@ Or, you can add Lift to your `pom.xml` like so:
+You can learn more about Maven integration [on the wiki](http://www.assembla.com/wiki/show/liftweb/Using_Maven).
 
-## Project Organization
+### Learning Lift
 
-The Lift Framework is divided into several components that are published independently. This organization enables you to use just the elements of Lift necessary for your project and no more.
+There are lots of resources out there for learning Lift. Some of our favorites include:
 
-### This Repository
+* [The Lift Cookbook](http://cookbook.liftweb.net/)
+* [Simply Lift](http://simply.liftweb.net)
 
-This repository, `framework`, contains the following components:
+If you've found one that you particularly enjoy, please open a PR to update this README!
 
-#### core
+## Issues & Pull Requests
 
-Core elements used by Lift projects. If you wish to reuse some of Lift's helpers and constructs, such as `Box`, this component may be all you need. However, a web application will most likely require one or more of Lift's components.
+Per our [contributing guidelines][contribfile], Issues on the Lift GitHub project are intended to
+describe actionable, already-discussed items. Committers on the project may open issues for
+themselves at any time, but non-committers should visit the [Lift mailing
+list](https://groups.google.com/forum/#!forum/liftweb) and start a discussion
+for any issue that they wish to open.
+
+We will accept issues and pull requests into the Lift codebase if the pull requests meet the following criteria:
+
+* The request is for a [supported version of Lift][supfile]
+* The request adheres to our [contributing guidelines][contribfile], including having been discussed
+  on the Mailing List if its from a non-committer.
+
+[contribfile]: https://github.com/lift/framework/blob/master/CONTRIBUTING.md
+[sigfile]: https://github.com/lift/framework/blob/master/contributors.md
+
+## Project Organization
 
-#### web
+The Lift Framework is divided into several components that are published independently. This organization enables you to use just the elements of Lift necessary for your project and no more.
 
-This component includes all of Lift's core HTTP and web handling. Including `lift-webkit` in your build process should be sufficient for basic applications and will include `lift-core` as a transitive dependency.
+### This Repository
 
-#### persistence
+This repository, `framework`, contains the following components:
 
-This component includes Mapper and Record, Lift's two ORMs. While you needn't use either and can use the ORM of your choice, Mapper and Record integrate nicely with Lift's idioms. Mapper is an ORM for relational databases, while Record is a broader ORM with support for both SQL databases and NoSQL datastores.
+* **core:** Core elements used by Lift projects. If you wish to reuse some of Lift's helpers and
+constructs, such as `Box`, this component may be all you need. However, a web application will most
+likely require one or more of Lift's components.
```

Comment:
This sentence seems a bit tautological. Do you mean "require one or more of Lift's *other components"?